.github/actions/cw-build-status/README.md

```diff
@@ -0,0 +1,84 @@
+# CW Build Status Github Action
+
+This action was originally ported over from [rosbag-uploader-ros1/cw-build-status][cw-build-status].
```

Comment:
Could you clarify the intent for this statements?
A few questions we should answer: Should fixes be sent there? Is this a 1:1 copy of this code? How do we keep those in sync? Should we?

Reply:
Removed since we won't be using that repo (plus it's private at the moment).
Regarding having a 1:1 copy of the code: no, we'll be moving this to it's own repo once we get it approved from OSS.
In the meantime, all issues related to this action will be part of colcon-bundle.